core/iwasm/compilation/aot_compiler.c

```diff
@@ -217,6 +217,74 @@ store_value(AOTCompContext *comp_ctx, LLVMValueRef value, uint8 value_type,
         return false;
     }
 
+    LLVMSetAlignment(res, 4);
+
+    return true;
+}
+
+static bool
+store_ref(AOTCompContext *comp_ctx, uint32 ref, LLVMValueRef cur_frame,
```

Comment:
Who should be responsible for committing parameters during invoking import function?
Seems the callee won't do this, should we commit the values when calling an import function?

Reply:
This is handled in aot_alloc_frame by setting the ref arguments' ref flags to 1，and then they are unchanged, no need to commit again. See the changes in aot_alloc_frame and llvm_jit_alloc_frame.

Reply:
The ref arguments are added to rootset when traversing aot stack frame during GC reclaim.

Reply:
But this only mark the ref flags, the actual value is not committed to the frame?
I think we should call `aot_gen_commit_values` at https://github.com/bytecodealliance/wasm-micro-runtime/blob/943274e3b90bba1fe2ff34281793b41c3f8b4b7c/core/iwasm/compilation/aot_emit_function.c#L654-L659

otherwise the frame for the native function may contain some dirty value from previous frames

Reply:
OK, that's really an issue.